src/NexusMods.App.UI/Pages/ILibraryDataProvider.cs

```diff
@@ -15,12 +17,31 @@ namespace NexusMods.App.UI.Pages;
 public interface ILibraryDataProvider
 {
     IObservable<IChangeSet<CompositeItemModel<EntityId>, EntityId>> ObserveLibraryItems(LibraryFilter libraryFilter);
+
+    IObservable<int> CountLibraryItems(LibraryFilter libraryFilter);
```

Comment:
So I think this might be returning the count of the roots, because if I download a new version of an existing item or another file of a mod page for which I already have a download, the count doesn't seem to change

Reply:
Yup, both the library and data provider `Count*` methods return the counts of just the parents.

Reply:
We can add an argument in a later PR to specify whether to get the counts for just the parents or for everything.